plugins/inventory/vmware_vm_inventory.py

```diff
@@ -215,6 +215,21 @@
     filters:
     - config.guestId == "rhel7_64Guest" or config.name == "rhel_20_04_empty"
 
+# Filter VM's based on regex conditions
+    plugin: community.vmware.vmware_vm_inventory
+    strict: False
+    hostname: 10.65.223.31
+    username: administrator@vsphere.local
+    password: Esxi@123$%
+    validate_certs: False
+    properties:
+    - 'config.name'
+    - 'config.guestId'
+    - 'guest.ipAddress'
+    - 'summary.runtime.powerState'
+    filters:
+    - guest.ipAddress is defined and guest.ipAddress is match('192.168.*') or guest.ipAddress is match('192.169.*')
```

Comment:
```suggestion
    - (guest.ipAddress is defined and guest.ipAddress is match('192.168.*')) or (guest.ipAddress is defined and guest.ipAddress is match('192.169.*'))
```

Reply:
Otherwise, I get 

```
Could not evaluate guest.ipAddress is defined and guest.ipAddress is match('192.168.*') or
guest.ipAddress is match('192.169.*') as host filters - 'guest' is undefined
```

Reply:
Thanks for checking on that Abhijeet. Can we use this 

```
- guest.ipAddress is defined and (guest.ipAddress is match('192.168.*') or guest.ipAddress is match('192.169.*'))
```
I tested it with a sync and it seems to be working fine.